### ISSUE TITLE
[YouTube Downloader] Output format mp4 is not available

### ISSUE THREAD

**Issue by shwezhu** [raycast/extensions#12110]
### Extension

https://www.raycast.com/vimtor/youtube-downloader

### Description

Error: Output format mp4 is not available

capabilities.js:589:20

---
586:           }, []);
587: 
588:         if (unavailable.length === 1) {
589:           return cb(new Error('Output format ' + unavailable[0] + ' is not available'));
590:         } else if (unavailable.length > 1) {
591:           return cb(new Error('Output formats ' + unavailable.join(', ') + ' are not available'));
592:         }
---

T:index.js:46:13445
I:index.js:46:13524
index.js:43:11429

### Steps To Reproduce

Download youtube video on Mac

**Comment by pickuse2013** [comment]
Hello @shwezhu, I encountered the same issue as you did. After I forcefully installed ffmpeg@6 and linked it, everything started working perfectly.

It seems that the problem was due to my homebrew having a newer version of ffmpeg installed, which was causing a conflict with this extension.

To resolve this issue, the following commands should help you:
```
brew install ffmpeg@6
brew unlink ffmpeg
brew link ffmpeg@6
```
![Snipaste_2024-05-02_12-03-28](https://github.com/raycast/extensions/assets/26698828/943cfbe1-ad65-445e-a0cb-3d965123df59)
